go.mod

```diff
@@ -5,8 +5,8 @@ go 1.16
 require (
 	github.com/gin-contrib/cors v1.3.0
 	github.com/gin-gonic/gin v1.7.1
-	github.com/iver-wharf/wharf-api-client-go v1.4.0
```

Comment:
Why downgrade?

Reply:
I didn't see a changelog entry for it and thought I had accidentally upgraded it.
Updated to v1.4.0 again.